srcpkgs/MLV-App/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'MLV-App'
+pkgname=MLV-App
+version=1.13
+revision=1
+wrksrc=${pkgname}-QTv${version}
+build_wrksrc="platform/qt"
+build_style=qmake
+hostmakedepends="qt5-qmake pkg-config qt5-host-tools"
+makedepends="qt5-multimedia-devel qt5-devel libgomp-devel ffmpeg"
```

Comment:
should ffmpeg be in `depends=`?

Reply:
By default, it installs a static binary of FFmpeg, but seems to work fine with regular FFmpeg.

The AUR package build does also not install this binary.